lib/hammer_cli_import/contenthost.rb

```diff
@@ -129,8 +129,8 @@ def post_import(_file)
         Dir.mkdir dir_name
         CSVHelper.csv_write_hashes(
           File.join(dir_name, "system-id_to_uuid-#{now.to_i}.map"),
-          [:system_id, :uuid, :org_id],
-          @map.sort_by { |x| [x[:org_id], x[:system_id], x[:uuid]] })
+          [:host_id, :uuid, :org_id],
```

Comment:
I thought the intent was system_id represented the Spacewalk/Sat 5 system_id and not the Foreman host_id ?